src/Coriolis/hydrostatic_spherical_advection_like_coriolis.jl

```diff
@@ -3,7 +3,13 @@
 # or "conservation form" advection scheme (which does not currently exist for
 # curvilinear grids).
 
-using Oceananigans.Advection: AbstractAdvectionScheme, _left_biased_interpolate_yᵃᶜᵃ, _right_biased_interpolate_xᶜᵃᵃ
+using Oceananigans.Advection: 
```

Comment:
Just curious: why put this in a separate file rather than adding to `hydrostatic_spherical_coriolis.jl`?